### PR TITLE
Dev-4510 - whitespace problem

### PR DESCRIPTION
**Pull request by quipo**
Also, restoring dep on main "PuerkitoBio/goquery" repo

### CODE REVIEW

**Comment by nicolaasuni** [comment]
Note: @quipo I have improved the GoOse library and merged it back to the original https://github.com/advancedlogic/GoOse , so you may want to consider to use this directly or re-sync the changes and also send a PR.

File: crawler.go [hunk 1]
```diff
@@ -40,6 +40,8 @@ func (c Crawler) Crawl() *Article {
 		return article
 	}
 
+	c.RawHTML = c.addSpacesBetweenTags(c.RawHTML)
```
Comment (nicolaasuni) [inline crawler.go:43]: does this change also requires a change in the tests?

Comment (quipo) [inline crawler.go:43]: I updated the tests

**Comment by nicolaasuni** [comment]
looks good to me

**Comment by quipo** [comment]
thanks @nicolaasuni

**Pull request merged**